common/beerocks/bwl/nl80211/sta_wlan_hal_nl80211.h

```diff
@@ -31,6 +31,7 @@ class sta_wlan_hal_nl80211 : public base_wlan_hal_nl80211, public sta_wlan_hal {
     sta_wlan_hal_nl80211(const std::string &iface_name, hal_event_cb_t callback);
     virtual ~sta_wlan_hal_nl80211();
 
+    virtual bool wps_pbc_multiap() override;
```

Comment:
Why the `_multiap` bit? Yes, we have to add the `multi_ap=1` bit to the command, but in prplmesh we're never going to use anything other than multi_ap, right?

Reply:
Right.